hephaistus/main-theme/distribution-system/chord/playground/nullist0/lib/src/main/kotlin/org/lia/dht/nullist0/chord/model/ChordScope.kt

```diff
@@ -0,0 +1,37 @@
+package org.lia.dht.nullist0.chord.model
+
+/**
+ * An interface to build identifier of Chord Ring.
+ */
+abstract class ChordScope<Id, Value> {
```

Comment:
What about `ChordIdentifier`?

Reply:
This class will provide the functionality about the evaluation of identifier of chord rings. I think the `Identifier` seems to create `Id` of chord ring, not other evaluations like `compareTo`, `plus`, or interval functions. As you mentioned, the `Scope` is not good to denote the functionality. I will change the class name.

Reply:
I renamed it at a5d769598f048ef3402ea544d851b656e015221e.

Reply:
Yeah, it was agonized one to me. Actually I want to suggest replace `Id` to `ChordIdentifier` 😄 Anyway, current it also LGTM